addon/v1alpha1/types_addontemplate.go

```diff
@@ -155,19 +155,17 @@ type CustomSignerRegistrationConfig struct {
 	Subject *Subject `json:"subject,omitempty"`
 
 	// SigningCA represents the reference of the secret on the hub cluster to sign the CSR
+	// the secret name must be "ocm-addon-template-signer", and the secret type must be "kubernetes.io/tls"
 	// +kubebuilder:validation:Required
 	SigningCA SigningCARef `json:"signingCA"`
```

Comment:
@qiujian16 since we removed the secret name field, do you think the SigningCARef is still necessary? or we just change it to signingCANamespace?

Reply:
I am thinking whether it makes more sense to watch secret in a certain namespace, rather than the a fixed name.